be/log.c

```diff
@@ -934,6 +940,17 @@ static int be_log_header_write(struct m0_be_log            *log,
 	return M0_RC(rc);
 }
 
+M0_INTERNAL int be_log_header_inc_generation(struct m0_be_log            *log,
```

Comment:
Usually, we name a function as:
xxx_noun_verb().

So, in this case, we prefer:
be_log_header_generation_inc()

Reply:
Done.